testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/DomainAdjuster.java

```diff
@@ -272,4 +264,8 @@ private void removeSubsystemExtensionIfExist(DomainClient client, PathAddress su
             // ignored, the extension does not exist
         }
     }
+
+    protected void adjustXpExtensions(DomainClient client, PathAddress profileAddress) throws Exception {
```

Comment:
Call this adjustExpansionExtensions.

XP is a Red Hat product term, and while there are lots of those in this module in this case its simple to not use that.